
/**: #no need to provide the node name, common param file for all instances
  ros__parameters:
    active_echos: 0
    add_transform_check_dynamic_updates: true
    add_transform_xyz_rpy: 0,0,0,0,0,0
    ang_res: 0.0
    auto_reboot: false
    client_authorization_pw: F4724744
    cloud_output_mode: 0
    cloud_topic: /cloud
    device_number: 0
    diagnostic_updater:
      period: 1.0
    emul_sensor: false
    encoder_mode: -1
    expected_frequency_tolerance: 0.1
    frame_id: laser_link
    hostname: 192.168.0.2
    imu_enable: true
    imu_frame_id: imu_link
    intensity: true
    intensity_resolution_16bit: true
    locationName: not defined
    max_ang: 2.356194490192345
    message_monitoring_enabled: true
    min_ang: -2.356194490192345
    min_intensity: 0.0
    nav_tf_child_frame_id: nav
    nav_tf_parent_frame_id: map
    new_IP_address: ''
    nodename: sick_scan #namespace for topic names 
    ntp_server_address: ''
    operationHours: 1003.3000000000001
    port: '2112'
    powerOnCount: 322
    publish_datagram: false
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    range_filter_handling: 0
    range_max: 100.0
    range_min: 0.0
    read_timeout_millisec_default: 5000
    read_timeout_millisec_kill_node: 150000
    read_timeout_millisec_startup: 120000
    ros_qos: -1
    scan_freq: 0.0
    scandatacfg_timingflag: -1
    scanner_type: sick_tim_5xx #Sick sensor product series no. 
    skip: 0
    slam_bundle: false
    slam_echo: ''
    start_services: true
    subscribe_datagram: false
    sw_pll_only_publish: true
    time_increment: -1.0
    time_offset: 0.0
    timelimit: 5
    use_binary_protocol: false
    use_generation_timestamp: true
    use_sim_time: false
